lib/rubocop/cop/rspec/stubbed_mock.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Checks that message expectations do not have a configured response.
+      #
+      # @example
+      #
+      #   # bad
+      #   expect(foo).to receive(:bar).with(42).and_return("hello world")
+      #
+      #   # good (without spies)
+      #   allow(foo).to receive(:bar).with(42).and_return("hello world")
+      #   expect(foo).to receive(:bar).with(42)
+      #
+      class StubbedMock < Base
+        MSG = 'Prefer `%<replacement>s` to `%<method_name>s` when ' \
```

Comment:
We usually use “Prefer foo *over* bar” in the offense messages.
```suggestion
        MSG = 'Prefer `%<replacement>s` over `%<method_name>s` when ' \
```